src/index.ts

```diff
@@ -71,7 +73,15 @@ export const handleTokenRequest = async (ctx: Context, request: Request) => {
 	);
 	const domain = new URL(request.url).host;
 	const issuer = new Issuer(domain, sk, pk);
-	const signedToken = await issuer.issue(tokenRequest);
+	let signedToken: TokenResponse;
+	if (ctx.env.EXPERIMENTAL_RSABSSA === 'true') {
+		const blindSignature = new Uint8Array(
+			await crypto.subtle.sign('RSA-RAW', sk, tokenRequest.blindedMsg)
```

Comment:
The `issue` function seems to do more than just signing, should that be replicated?

https://github.com/cloudflare/privacypass-ts/blob/755dbc7c719c530030f612f924573efe0eb9f511/src/priv_verif_token.ts#L187-L203

Reply:
pp-issuer only supports Type 2 (public) tokens. The `issue` function is the following https://github.com/cloudflare/privacypass-ts/blob/755dbc7c719c530030f612f924573efe0eb9f511/src/pub_verif_token.ts#L177, which just does signing.